From 8eda468cb9445e5a9e2505d47fc5b15eae453a0a Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 22 Sep 2017 20:23:58 +0200
Subject: [PATCH] def: add new constant LONG_LINE_MAX

LONG_LINE_MAX is much like LINE_MAX, but longer.

As it turns out LINE_MAX at 4096 is too short for many usecases. Since
the general concept of having a common maximum line length limit makes
sense let's add our own, and make it larger (1MB for now).

(cherry picked from commit 189912440f6545404e84b3cd1d6ca54f1057e3e6)
---
 src/basic/def.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/basic/def.h b/src/basic/def.h
index b1a3bc190b..e2d91519ba 100644
--- a/src/basic/def.h
+++ b/src/basic/def.h
@@ -86,3 +86,5 @@
         "/usr/local/lib/" n "\0"                \
         "/usr/lib/" n "\0"                      \
         _CONF_PATHS_SPLIT_USR(n)
+
+#define LONG_LINE_MAX (1U*1024U*1024U)
